src/scenic/core/pruning.py

```diff
@@ -48,10 +74,17 @@ def matchPolygonalField(heading, position):
     bounded disturbance. Returns a triple consisting of the matched field if
     any, together with lower/upper bounds on the disturbance.
```

Comment:
Need to update docstring (now matches `yaw` attributes of orientations defined by...).